### PR TITLE
Fixed issues with macOS Privacy Settings (Camera and Microphone)

### PR DESCRIPTION
**Pull request by RishabSwift**
This looks at issue #1128,  #1128,  #1082, #1361 and so many more..

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "caprine",
 	"productName": "Caprine",
-	"version": "2.46.0",
+	"version": "2.46.1",
```
Comment (sindresorhus) [inline package.json:4]: You should never bump a version in a PR. It's up to the maintainers to do this.

File: build/entitlements.mas.plist [hunk 2]
```diff
@@ -10,5 +10,9 @@
 	<true/>
 	<key>com.apple.security.files.downloads.read-write</key>
 	<true/>
+	<key>com.apple.security.device.microphone</key>
+    	<true/>
+    	<key>com.apple.security.device.audio-input</key>
+    	<true/>
```
Comment (sindresorhus) [inline build/entitlements.mas.plist:16]: Wrong indentation.

**Comment by sindresorhus** [comment]
You also need to set usage descriptions for these.

File: build/entitlements.mas.plist (hunk 2 above)
Comment (RishabSwift) [reply build/entitlements.mas.plist:16]: What do you mean?

Comment (sindresorhus) [reply build/entitlements.mas.plist:16]: You're mixing space and tab indentation.

**Comment by sindresorhus** [comment]
https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/requesting_authorization_for_media_capture_on_macos

File: package.json (hunk 1 above)
Comment (sindresorhus) [reply package.json:4]: You could do this yourself...

**Comment by kopach** [comment]
Hi @sindresorhus. What's the status of this? When this can be merged? I can't wait to have this issue finally fixed :)

**Comment by sindresorhus** [comment]
@kopach It think it's very clear from the comments here that this PR still has one outstanding task before it's mergeable.

**Pull request closed without merging**